globus_cli/commands/endpoint/search.py

```diff
@@ -26,6 +26,9 @@ def endpoint_search(filter_fulltext, filter_owner_id, filter_scope):
     """
     Executor for `globus endpoint search`
     """
+    if filter_scope == 'all' and not filter_fulltext:
+        raise click.UsageError('filter_scope "all" requires a search filter.')
```

Comment:
Yeah, I went with this one.

You don't get whatever fancy "short" line-wrapping goes on with other things (e.g. help output).

I force pushed an update if you want to re-pull and see for yourself.